website/src/components/Header.vue

```diff
@@ -78,19 +77,12 @@
 
 <script>
 export default {
-  props: {
-    showToggle: {
-      type: Boolean,
-      default: false
-    }
-  },
-
   methods: {
-    toggleSidebar() {
-      if (document.body.classList.contains('show-sidebar')) {
-        document.body.classList.remove('show-sidebar')
+    toggleLeftbar() {
+      if (document.body.classList.contains('show-leftbar')) {
+        document.body.classList.remove('show-leftbar')
       } else {
-        document.body.classList.add('show-sidebar')
+        document.body.classList.add('show-leftbar')
```

Comment:
Why not `document.body.classList.toggle('show-leftbar')`

Reply:
Oh I didn't know about that 😂